README.md

```diff
@@ -149,7 +149,8 @@ To run tests, run the following command:
 
 You can deploy this project using:
 
-[]comming soon
+ This link: 
+(https://axcell-0.github.io/portfolio-setup/)
```

Comment:
- [ ] [OPTIONAL] I think it would be better if you add this live demo link under the LIVE DEMO section on your Readme :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230896639-020ed556-a1ed-47ce-bcd3-ed0708c7268e.png)
 as that would be a more appropriate place for the Live link, here you can say it is deployed with github pages :+1: